src/JustSaying/IPublishConfiguration.cs

```diff
@@ -8,5 +8,6 @@ public interface IPublishConfiguration
     int PublishFailureReAttempts { get; set; }
     TimeSpan PublishFailureBackoff { get; set; }
     Action<MessageResponse, Message> MessageResponseLogger { get; set; }
+    Action<MessageBatchResponse, IEnumerable<Message>> MessageBatchResponseLogger { get; set; }
```

Comment:
This is a binary-breaking change. Can it be broken out to not be so?

Reply:
The response logger is legacy and its job is better performed by middleware now, so would suggest leaving this out and maybe even marking the old response logger as obsolete 👍.